frontend/src/components/atoms/TableHeaderCell/TableHeaderCell.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const Container = styled.div`
+  margin: 0 8px;
+  padding: 8px;
+  
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  
+  color: ${({ color, theme }) => {
```

Comment:
Move this function to utils module. This function can be used in different places.

Reply:
moved